### ISSUE TITLE
Running DeepLab on PASCAL VOC 2012 Semantic Segmentation Dataset

### ISSUE THREAD

**Issue by betterhalfwzm** [tensorflow/models#3633]
ubuntu 16.04
cuda 9.0 
cudnn 7.04
tf 1.6
when i used the trained model  to visualize the images ,the images of result  are all the black ,can you solve it ?  The model is trained on my machine

**Comment by toby-coleman** [comment]
I'm seeing a similar problem when I try to do inference using DeepLab.  What I've found is:
* Inference using the frozen graph inside [deeplabv3_pascal_train_aug_2018_01_04.tar.gz](http://download.tensorflow.org/models/deeplabv3_pascal_train_aug_2018_01_04.tar.gz) works fine; but
* If I use `export_model.py` to create my own frozen graph from the model checkpoint instead, then all pixels get classified as background.

**Comment by betterhalfwzm** [comment]
The model is trained on my machine@toby-coleman

**Comment by betterhalfwzm** [comment]
The model is trained on my machine @ toby-coleman

**Comment by aquariusjay** [comment]
Could you please (1) make sure all the flag values are the same for {train,eval,vis,export} and maybe (2) `sh local_test.sh` and see if you can pass the simple test?

**Comment by betterhalfwzm** [comment]
@aquariusjay 
(1)All the flag are the same .I set fine_tune_batch_norm = False  learning rate=0.0001 and 0.00001
(2)Ran 5 tests in 15.637s
    OK

**Comment by aquariusjay** [comment]
Please report the detailed results after `sh local_test.sh`.
1. What is the evaluation performance you got?
2. Have you observed any visualization results under the folder, `segmentation_results`?

In addition to make sure the flags are the same, please also make sure the flag, tf_initial_checkpoint, is set correctly.

**Comment by betterhalfwzm** [comment]
@aquariusjay After your reminding, I found the mistake.I used the wrong tf_initial_checkpoint.Thank you very much.

**Comment by aquariusjay** [comment]
Good job on figuring out the problem and glad to know that the issue is resolved. Closing this issue.

**Comment by Dachaser** [comment]
@betterhalfwzm  whats was the problem with tf_initial_checkpoint??? because me too use export_model.py to create my own frozen graph from the model checkpoint and then all pixels get classified as background in the deeplab_demo.ipynb.
